README.md

```
@@ -17,17 +20,19 @@ yarn dev
 ```
 
 ### Environment variables
+
 | Name                    | Description                                         | Recommended development value                                                                                                                                   |
 | ----------------------- | --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `APP_ENV`               | Makes the favicon red when not set to `prod`.       | `dev`                                                                                                                                                           |
 | `DISCORD_CLIENT_ID`     | Discord application client ID, used for OAuth2.     | Create a Discord Application, register `http://localhost:3000/api/auth/callback/discord` as an OAuth2 callback URL, and paste the application's client ID here. |
 | `DISCORD_CLIENT_SECRET` | Discord application client secret, used for OAuth2. |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)